wled00/wled.h

```diff
@@ -903,7 +903,12 @@ WLED_GLOBAL bool ledStatusState _INIT(false); // the current LED state
 #endif
 
 // server library objects
+#if defined(CONFIG_IDF_TARGET_ESP32C6)
+// ESP32-C6 uses patched ESPAsyncWebServer fork with simpler constructor
```

Comment:
@willmmiles what do you think - is this specific to C6 or maybe specific for ESP-IDF v5?